dart/shelf/pubspec.yaml

```
@@ -0,0 +1,8 @@
+name: shelf_app
+
+environment:
+  sdk: ^3.0.0
+
+dependencies:
+  shelf: 1.4.1
```

```suggestion
  shelf: ">=1.4.1 <1.5.0"
```
or maybe
```suggestion
  shelf: "^1.4.1"
```
not sure which is the common usage in dart